rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -54,35 +54,37 @@ def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False,
         self.reverse_channels = reverse_channels
         self.tensor_dtype = tensor_dtype
         self.batch_size = self.loader._batch_size
-        self.w = self.loader.getOutputWidth()
-        self.h = self.loader.getOutputHeight()
-        self.n = self.loader.getOutputImageCount()
         self.rim = self.loader.getRemainingImages()
         self.display = display
         self.iter_num = 0
         self.sequence_length = sequence_length
         print("____________REMAINING IMAGES____________:", self.rim)
-        color_format = self.loader.getOutputColorFormat()
-        self.p = (1 if color_format is types.GRAY else 3)
-        self.out = np.empty(
-                (self.batch_size*self.n,int(self.h/self.batch_size), self.w,self.p), dtype="ubyte")
+        self.output = self.dimensions = self.dtype = None
 
     def next(self):
         return self.__next__()
 
     def __next__(self):
-        self.iter_num +=1
         if(self.loader.isEmpty()):
             raise StopIteration
-        if self.loader.run() != 0:
+
+        if self.loader.rocalRun() != 0:
             raise StopIteration
+        else:
+            self.output_tensor_list = self.loader.getOutputTensors()
+        self.iter_num +=1
         #Copy output from buffer to numpy array
-        self.loader.copyImage(self.out)
-        img = torch.from_numpy(self.out)
+        if self.output is None:
+            self.dimensions = self.output_tensor_list[0].dimensions()
+            self.dtype = self.output_tensor_list[0].dtype()
+            self.layout = self.output_tensor_list[0].layout()
+            self.output = np.empty((self.dimensions[0]*self.dimensions[1], self.dimensions[2], self.dimensions[3], self.dimensions[4]), dtype = self.dtype)
```

Comment:
Please add space before and after operators
Applicable to all such instances in this file